src/store.rs

```diff
@@ -404,7 +404,7 @@ impl<F: LurkField> ContPtr<F> {
 pub struct RawPtr<F: LurkField>(isize, PhantomData<F>);
 
 impl<F: LurkField> RawPtr<F> {
-    fn new(p: usize) -> Self {
+    pub(crate) fn new(p: usize) -> Self {
```

Comment:
`pub(crate)` explicitly excludes the possibility of importing `lurk-rs` as a library. Are we sure we want that constraint?

Reply:
Do you mean as opposed to `pub`, or as as opposed to it being private like it was before? How does it preclude use as a library? Do you mean a library that allows for manipulation of `RawPtr`s (which I wouldn't assume `lurk-rs` as a library would do. For example, `fcomm` does this (use `lurk-rs` as a library) and hasn't been harmed by this.

I'm open to making this `pub` if there's a good reason for it (and if that's what you're suggesting), but it's not clear to me that it should be exposed. It's an internal detail, really, in my eyes.

Reply:
Yes, generally I would prefer making things just `pub` vs `pub(crate)`, because I wouldn't want to assume up-front that `fcomm` or another downstream lib would ever want to use this.

But it's a minor quibble, doesn't affect anything now and easily fixable if need be in future.